src/lsst/cm/tools/core/panda_utils.py

```diff
@@ -233,6 +228,35 @@ def determine_error_handling(dbi: DbInterface, errors_agg: dict, max_pct_failed:
     return panda_status
 
 
+# map to take the many statuses and map them to end results
```

Comment:
Are random enums okay to float in the middle of files? Better that it's not needlessly in a function but does it have to go at the top?? @fritzm

Reply:
To be clear Fritz, I learned how to program by StackExchange in grad school and code like an absolute monster so I would like to know best practices here!

Reply:
(And, FWIW, style-wise, I think having the enum defined close to the function which accepts/uses it is fine and might actually be better than having it far separated in the file.  I tend to favor optimizing readability without having to scroll around too far to things you can't see.  YMMV...)

Reply:
Yeah, but I wouldn't worry it until maybe the next time you have to edit that enum anyway?  It seems perfectly clear what the current code means.  We all knew it was an enum.

Reply:
Gotcha, it'll get edited in the next pass.